### ISSUE TITLE
Typos in Logistic chapter  

### ISSUE THREAD

**Issue by ilapros** [daviddalpiaz/appliedstats#94]
I am reading through the chapter and found some typos/inconsistencies - sorry for not making a pull request (and thanks for making this great book available!)

When introducing the sigmoid function you have

\text{logit}^{-1}(\xi) = \frac{e^\xi}{1 + e^{\xi}} = \frac{1}{1 + e^{-\xi}}

it should be 

\text{logit}^{-1}(\xi) = \frac{e^\xi}{1 + e^{\xi}} =  \frac{1}{1 + e^{-\xi}}-1

I believe. 

Some lines above you have

(Note, many more machine learning focused texts will use  p   as the number of parameters. This is an arbitrary choice, but you should be aware of it.)

but you are also using p as the number of parameters, so I was confused by the sentence.

**Comment by daviddalpiaz** [comment]
I'm not sure I understand your suggested revision to the sigmoid function. My definition seems to match what is seen here: https://en.wikipedia.org/wiki/Sigmoid_function (Perhaps I am missing something?)

To you other point, yes that is a typo. It should say number of predictors. Fixing now.

**Comment by ilapros** [comment]
Yes sorry - I hadn't notice the change in the sign of ^{x} - my "correction" would have been \frac{1}{1 + e^{\xi}}-1, my bad.
